spec/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SessionsController, type: :controller do
```

Comment:
We are missing a test to create a new session (sign in). The http success status only indicates you were able to get / access the form / new page successfully, but it doesn't indicate that a successful sign in.

[`REQUIRED`]